05week/checkers.js

```diff
@@ -10,11 +10,41 @@ const rl = readline.createInterface({
 
 function Checker() {
   // Your code here
+  const player1="B";
+  const player2="R";
+  const changePlayer=()=>{
+    	if (player=="B")
+    		player="R";
+    	else
+    		player="B";
+    }
+
+  const isValidMove=(whichPiece, toWhere)=>{
+    if(whichPiece.symbol=player1){
+
+    }
+
+  }
+
+
+const whichPiece=Board.grid[row][column].symbol.onclick
+
+}
+const toWhere=Board.grid[row][column].onclick
```

Comment:
This game is being designed for the console.  OnClick methods are typically used when designing web browser applications